app/src/main/java/org/mate/ui/EnvironmentManager.java

```diff
@@ -19,6 +19,7 @@
     public static final String ACTIVITY_UNKNOWN = "unknown";
     private static final String DEFAULT_SERVER_IP = "10.0.2.2";
     private static final int DEFAULT_PORT = 12345;
+    //private static final String DEFAULT_SERVER_IP = "localhost";
     //private static final String DEFAULT_SERVER_IP = "192.168.1.26";
     private static final String METADATA_PREFIX = "__meta__";
     private static final String MESSAGE_PROTOCOL_VERSION = "1.1";
```

Comment:
Protocol version should be bumped to 1.2 to reflect addition of new command. The same should be done in the MATE-server. A warning will be printed if the version numbers mismatch.

Reply:
done